sqlstore/src/main/java/uk/ac/bbsrc/tgac/miso/persistence/impl/HibernatePoolableElementViewDao.java

```diff
@@ -0,0 +1,71 @@
+package uk.ac.bbsrc.tgac.miso.persistence.impl;
+
+import java.util.Collections;
+
+import org.hibernate.Criteria;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.criterion.Restrictions;
+import org.hibernate.type.LongType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+import org.springframework.transaction.annotation.Transactional;
+
+import uk.ac.bbsrc.tgac.miso.core.data.impl.view.PoolableElementView;
+import uk.ac.bbsrc.tgac.miso.core.data.type.PlatformType;
+import uk.ac.bbsrc.tgac.miso.persistence.PoolableElementViewDao;
+
+@Transactional(rollbackFor = Exception.class)
+@Repository
+public class HibernatePoolableElementViewDao implements PoolableElementViewDao, HibernatePaginatedDataSource<PoolableElementView> {
+
+  // Make sure these match the HiberateLibraryDilutionDao
+  private static final String[] SEARCH_PROPERTIES = new String[] { "dilutionName", "dilutionBarcode", "libraryName", "libraryAlias",
+      "libraryDescription" };
+
+  @Autowired
+  private SessionFactory sessionFactory;
+
+  @Override
+  public Session currentSession() {
+    return sessionFactory.getCurrentSession();
+  }
+
+  @Override
+  public String getProjectColumn() {
+    return "projectId";
+  }
+
+  @Override
+  public Class<? extends PoolableElementView> getRealClass() {
+    return PoolableElementView.class;
+  }
+
+  @Override
+  public Iterable<String> listAliases() {
+    return Collections.emptyList();
+  }
+
+  @Override
+  public String propertyForSortColumn(String original) {
+    return original;
+  }
+
+  @Override
+  public void setPlatformType(Criteria criteria, PlatformType platformType) {
```

Comment:
I realize this is for the PaginatedDataSource, but `dao.setPlatformType` and similar are confusing method names for what they do. Nothing to fix in this PR; just wanted to mention it